src/models/bonw16/repairs.md

```diff
@@ -1,3 +1,283 @@
+2. Pull the speaker(s) off of the plastic posts.
+
+![Audio daughterboard](./img/speakers-left-right.webp)
+
+3. Disconnect the speaker connector(s) from the motherboard.
+    - The left speaker (pictured on the right above) connects to `J_SPK1`.
+    - The right speaker (pictured on the left) connects to `J_SPK2`.
+4. Replace the speaker(s) and bottom cover.
+
+## Replacing the cooling system:
+
+The Bonobo WS 16 has a single heatsink assembly with two fans. This assembly cools the CPU and GPU.
+
+If the fans become noisy and cleaning them out doesn't fix the issue, you may need a new fan. [Contact support](https://support.system76.com) to start a warranty claim or parts purchase.
+
+Depending on your climate and the age of the machine, it may be necessary to apply new thermal paste between the CPU/GPU and the heatsink. Thermal paste helps facilitate effective heat transfer between the CPU/GPU and the cooling equipment. These instructions can also be used in the unlikely event your heatsink needs to be replaced.
+
+**Part numbers:**
+- Heatsink/fan assembly: `6-31-X580N-101`
+- Individual fans (may not be available separately):
+    - Left fan: Yingfan `NA901212HHT4B10001 DX1`
+    - Right fan: Yingfan `NB901212HHT4B0001 DX2`
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 15 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+### Steps to replace the heatsink/thermal paste:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover) and [remove the SSD from slot 1](#replacing-an-m2nvme-ssd) (if installed).
+2. [Disconnect the subwoofer](#steps-to-replace-the-subwoofer) from the motherboard's `J_SPKL5` header.
+    - You can optionally remove the subwoofer. In the photo below, the subwoofer has been removed to better illustrate the location of the heatsink screws.
+3. Remove the fourteen heatsink and fan screws in order of the stamped numbers, starting with #1, then #2, and continuing until you have removed #14.
+    - Screws #1-#12 are held captive, and will not completely detatch from the heatsink/fans.
+    - Screws #13 and #14 are slightly smaller than the rest of the screws, and are not held captive.
+    - Do not remove the smaller screws holding the fan covers onto the fans.
+
+![Thermal screws](./img/thermal-system.webp)
+
+4. Remove any clear tape covering the fan wires.
+5. Unplug the black fan connectors from the motherboard's fan headers, `J_VGAFAN1` (highlighted red above) and `J_CPUFAN1` (highlighted green).
+6. Remove the heatsink/fans from the case, being careful not to bend the heatsink pipes. It may take some pressure to break the seal of the thermal paste.
+7. Using a paper towel, remove the existing thermal paste from the CPU, GPU, VRAM chips, and heatsink. You may also use a small amount of rubbing alcohol if the old paste is dried or difficult to remove.
```

Comment:
I believe what's used on the VRAM chips is a more delicate and less dense thermal pad, not liquid thermal paste like what is used on CPU/GPU dies directly. Are we sure we want to direct readers to remove these pads and use liquid paste on the VRAM chip packages? I imagine it'll work, but it seems like it could make quite a mess.

Reply:
All sections have been updated with new verbiage to explain the thermal putty. The bonw16 photos have been retaken to show the putty collection process.